PhysicsTools/NanoAOD/plugins/ProcessRefinedJets.cc

```diff
@@ -0,0 +1,145 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+#include "DataFormats/PatCandidates/interface/MET.h"
+#include "TLorentzVector.h"
+
+#include <cmath>
+
+//This producer was written to embed the final refined FastSim jet pT values
+//so that they can be used as input to the pT-based sorting routine, as well
+//as to implement the Type-1 MET correction based on the refined jets.
+class ProcessRefinedJets : public edm::stream::EDProducer<> {
```

Comment:
could be `edm::global::EDProducer<>`; member variables are not modified during `produce()`